contrib/parseq-lambda-names/README.md

```diff
@@ -0,0 +1,5 @@
+Parseq Lambda Names
+==========================
+
+This project aims to provide more meaningful default descriptions for Parseq tasks. Using ASM, it tries to locate where lambda expression is defined in source code and also infer some details about its execution like function call within lambda expression with number of arguments.
+
```

Comment:
Explain how this project is supposed to be used (include on a classpath) and that it is fragile and can break (generate different output) potentially between minor JVM versions.
Also, include info with what JVM versions this is working correctly.